otx/algorithms/classification/adapters/mmcls/models/heads/semisl_cls_head.py

```diff
@@ -1,12 +1,16 @@
+"""Module for defining semi-supervised learning for multi-class classification task."""
 # Copyright (C) 2022 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 #
+# pylint: disable=too-many-locals, too-many-arguments
```

Comment:
Please disabling pylint as minimum as possible. How about setting in on necessary functions or classes?